src/components/share/index.js

```diff
@@ -62,7 +64,30 @@ module.exports = {
             this.modalTitle = title;
             this.modalLinks = links;
         },
-
+        gatherAllUsersToUnshare: function(currentSharedWithUsernames, usernamesToUnshare) {
+            let friendGroupUid = this.getGroupUid(peergos.shared.user.SocialState.FRIENDS_GROUP_NAME);
+            let includesFriends = usernamesToUnshare.indexOf(friendGroupUid) > -1;
+            let followersGroupUid = this.getGroupUid(peergos.shared.user.SocialState.FOLLOWERS_GROUP_NAME);
+            let includesFollowers = usernamesToUnshare.indexOf(followersGroupUid) > -1;
+            let allUsers = usernamesToUnshare.concat([]);
+            if (includesFriends) {
+                for(var i = 0; i < currentSharedWithUsernames.length; i++) {
```

Comment:
i seems to be indexing two different things here

Reply:
Fixed